utils/build/virtual_machine/provisions/host-auto-inject-install-script/auto-inject_host_script.yml

```diff
@@ -7,4 +7,7 @@
         local_path: utils/build/virtual_machine/provisions/auto-inject/tracer_debug/debug_config.yaml
   remote-command: |
     bash execute_install_script.sh host 
-    sudo cp debug_config.yaml /etc/datadog-agent/inject/debug_config.yaml
\ No newline at end of file
+    sudo mkdir -p /var/run/datadog-installer
+    sudo mkdir -p /opt/datadog/apm/inject/run
```

Comment:
(adding ctx) this is required as long as we have the mix between deb/rpm tracers, injector OCI and agent 
On complete of the OCI migration we can remove it, including the config below